README.md

```diff
@@ -51,4 +51,4 @@ In this module you will learn about what are Crypto Wallets and how to download
   - [Trust Wallet](https://trustwallet.com/)
   - [Atomic Wallet](https://atomicwallet.io/)
     
-   P.S. - To connect other wallets in LearnWeb3DAO you have to use [Wallet Connect](https://walletconnect.com/).Here is the list of all [wallets supported by WalletConnect](https://walletconnect.com/registry/wallets).
+   P.S. - To connect other wallets in LearnWeb3DAO you have to use [Wallet Connect](https://walletconnect.com/).Here is the list of all [wallets supported by WalletConnect](https://walletconnect.com/registry?type=wallet).
```

Comment:
Can you add your discord id?  I would like to give you the contributor role on discord 😄?

Reply:
Discord: SatvikG7#3180

Reply:
I cant find you on LearnWeb3's discord. Tag me so that I can give you the role

Reply:
I have DMed you just now.